.github/workflows/constraints.txt

```diff
@@ -1,3 +1,3 @@
-pip==23.0.1
+pip==24.0
```

Comment:
I got the error.

https://github.com/z3z1ma/dbt-osmosis/actions/runs/9402908605/job/25898108964